website/content/docs/commands/alloc/signal.mdx

```diff
@@ -31,15 +35,31 @@ allocation's namespace.
 ## Signal Options
 
 - `-s`: Signal to send to the tasks. Valid options depend on the driver.
+
+- `-task`: Specify the individual task that will receive the signal.
+
 - `-verbose`: Display verbose output.
 
 ## Examples
 
 ```shell-session
 $ nomad alloc signal eb17e557
 
-$ nomad alloc signal eb17e557 foo
-Could not find task named: foo, found:
+$ nomad alloc signal eb17e557 redis
+Could not find task named: redis, found:
 * test
 <blocking>
 ```
+
+Specifying task name with the `-task` option:
+
+```shell-session
+$ nomad alloc signal -task redis eb17e557
+```
+
+If task name is specified using both options, the command argument is ignored.
+The following will signal the redis task only, not the api task:
```

Comment:
Maybe quote these names to make it clear they're names and not part of the command?

```suggestion
If task name is specified using both options, the command argument is ignored.
The following will signal the "redis" task only, not the "api" task:
```

(Same for the other command docs)